gateway/src/main/java/io/camunda/zeebe/gateway/admin/backup/BackupRequestHandler.java

```diff
@@ -41,10 +41,7 @@ public CompletionStage<Long> takeBackup(final long backupId) {
                             .map(brokerClient::sendRequestWithRetry)
                             .toArray(CompletableFuture[]::new))
                     .thenApply(ignore -> backupId))
-        .exceptionallyCompose(
-            error ->
-                CompletableFuture.failedFuture(
-                    new BackupOperationFailedException("take", backupId, error.getCause())));
+        .exceptionallyCompose(error -> CompletableFuture.failedFuture(error.getCause()));
```

Comment:
:wrench: I think `BackupOperationFailedException` is unused now and could be removed

Reply:
I'm also not exactly sure what the point of unwrapping the exception here is. The endpoint that consumes the future still has code to handle a `CompletionException` by unwrapping the cause. I think we could just remove the `exceptionallyCompose` altogether.

Reply:
Yeah, good point :smile: